tests/shared/config/test_config.py

```diff
@@ -0,0 +1,46 @@
+import logging
+
+import pytest
+
+from iso15118.secc import Config
+from iso15118.shared.messages.enums import AuthEnum, Protocol
+
+
+class TestSECCConfig:
+    @pytest.fixture(autouse=True)
+    def setup_config(self):
+        self.config = Config(
+            iface="eth0",
+            log_level=logging.DEBUG,
+            enforce_tls=False,
+            free_charging_service=False,
+            free_cert_install_service=True,
+            allow_cert_install_service=True,
+            use_cpo_backend=False,
+            supported_protocols=[Protocol.ISO_15118_2, Protocol.DIN_SPEC_70121],
+            supported_auth_options=[AuthEnum.EIM],
+            standby_allowed=False,
+        )
+
+    @pytest.mark.parametrize(
+        "config_name, new_value",
+        [
+            ("iface", "en0"),
+            ("log_level", logging.INFO),
+            ("enforce_tls", True),
+            ("free_charging_service", True),
+            ("free_cert_install_service", False),
+            ("use_cpo_backend", True),
+            (
```

Comment:
Would our changes work for shared settings as well? Like PKI_PATH, MESSAGE_LOG_JSON etc?

Reply:
Done